### ISSUE TITLE
Envoy 1.28

### ISSUE THREAD

**Issue by sam-heilbron** [solo-io/gloo#8979]
### Gloo Edge Product

Open Source

### Gloo Edge Version

1.17

### Is your feature request related to a problem? Please describe.

I want Gloo Edge to use the latest stable Envoy version

### Describe the solution you'd like

- [ ] Walk through the Envoy release notes, understanding what has changed since 1.27 (https://www.envoyproxy.io/docs/envoy/v1.28.0/version_history/v1.28/v1.28.0)
- [ ] Write up a document that outlines the changes (including field deprecations, new features, breaking changes) that are relevant to the Gloo Edge product. In this document, propose which of these changes we will incorporate in this feature request, and perhaps a set of features that we should consider supporting in the future.
- [ ] Schedule a review of this plan, inviting members of the Edge team, @solo-io/gloo-edge-leads , get buy in on the plan.
- [ ] Execute the plan, following https://github.com/solo-io/gloo/issues/8514 and previous upgrades as inspiration
- [ ] Identify additional tests that will improve the stability of our Envoy integration, or provide a rationale for why that is not necessary
- [ ] Work with @solo-io/solo-docs to update our support matrix and any other relevant user-facing docs that are impacted by this change

**Comment by sam-heilbron** [comment]
This work is blocked by the data plane repositories supporting this version:
- [ ] https://github.com/solo-io/envoy-gloo-ee/issues/677

**Comment by nfuden** [comment]
Closing in favor of https://github.com/solo-io/gloo/issues/9232
